pkg/forwarder/forwarder.go

```diff
@@ -464,7 +464,8 @@ func (f *DefaultForwarder) SubmitAgentChecksMetadata(payload Payloads, extra htt
 
 // SubmitMetadata will send a metadata type payload to Datadog backend.
 func (f *DefaultForwarder) SubmitMetadata(payload Payloads, extra http.Header) error {
-	return f.submitV1IntakeWithTransactionsFactory(payload, extra, f.createHTTPTransactions)
+	transactions := f.createHTTPTransactions(v1MetadataEndpoint, payload, false, extra)
+	return f.sendHTTPTransactions(transactions)
```

Comment:
Is this correct, or do we want a config flag to allow these to still be sent to `/intake` by default?

Reply:
* metadata payloads other than the inventories payload should still go to `/intake` (but I haven't checked what payloads are sent with `SubmitMetadata`)
* I think we're fine without a flag for the inventories payload

Reply:
I guess the "metadata payloads other than inventories" are host metadata and agent checks metadata?  Those are still using `/intake` as desired.

Reply:
Correct, and yes these payloads should continue to be sent to `/intake`